docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -140,14 +160,14 @@ As outlined in the [Adding Add-on Packs](adding-add-on-packs.md) guide, you can
 
 ```bash
 .
-├── pack.json           # Mandatory
-├── values.yaml         # Mandatory
-├── manifests           # Mandatory
+├── pack.json           # Mandatory.
+├── values.yaml         # Mandatory.
+├── manifests           # Mandatory.
     ├── manifest-1.yaml
     ├── manifest-2.yaml
 │   └── manifest-3.yaml
-├── logo.png            # Mandatory
-└── README.md           # Optional
+├── logo.png            # Mandatory.
+└── README.md           # Optional.
 ```
```

Comment:
Disable copy functionality for this code snippet perhaps? https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1947435009/Docusaurus+Migration#hide-clipboard